client/src/modules/list/components/Items/ListItemDescription/ListItemDescription.jsx

```diff
@@ -0,0 +1,275 @@
+import React, { Fragment, PureComponent } from 'react';
+import PropTypes from 'prop-types';
+import { withRouter } from 'react-router-dom';
+import { connect } from 'react-redux';
+import _trim from 'lodash/trim';
+import _isEqual from 'lodash/isEqual';
+import Linkify from 'react-linkify';
+import Textarea from 'react-textarea-autosize';
+import classNames from 'classnames';
+
+import Preloader, {
+  PreloaderSize,
+  PreloaderTheme
+} from 'common/components/Preloader';
+import { RouterMatchPropType } from 'common/constants/propTypes';
+import { updateListItem } from '../model/actions';
+import { KeyCodes } from 'common/constants/enums';
+import { AbortPromiseException } from 'common/exceptions/AbortPromiseException';
+import { makeAbortablePromise } from 'common/utils/helpers';
+
+class ListItemDescription extends PureComponent {
+  pendingPromise = null;
+
+  constructor(props) {
+    super(props);
+
+    const { description } = this.props;
+    const trimmedDescription = description.trim();
+
+    this.state = {
+      isDescriptionUpdated: false,
+      descriptionInputValue: trimmedDescription,
+      isDescriptionEdited: false,
+      isFocused: false,
+      pending: false
+    };
+
+    this.descriptionTextarea = React.createRef();
+  }
+
+  componentDidUpdate(prevProps, prevState) {
+    this.isDescriptionUpdated();
+    const { isDescriptionEdited: isPreviousIsDescriptionEdited } = prevState;
+    const { isDescriptionEdited } = this.state;
+
+    if (!isPreviousIsDescriptionEdited && isDescriptionEdited) {
+      this.descriptionTextarea.current.focus();
+      this.handleFocus();
+    }
+
+    if (isPreviousIsDescriptionEdited && !isDescriptionEdited) {
+      this.handleBlur();
+    }
```

Comment:
please extract textarea field to separate component to avoid such things in component Did update

You will have 'focus() in componentDidMount()

Reply:
In my opinion, both solutions have their advantages and disadvantages. I tried to implement your solution, but to make it work exactly the same, I would need more time. I think it's better now to concentrate on the last task in the current sprint than to spend time getting the same effect in a different way (the current solution works correctly).